askui_example/logging/logger.ts

```diff
@@ -1,10 +1,28 @@
 import "@askui/jest-allure-circus";
+import * as os from 'os';
 
 declare const allure: {
     step(name: string, body?: () => void): void;
     attachment(name: string, content: string, type: string): void;
+    writeEnvironmentInfo(info: Record<string, string>): void;
 }
 
+// Generate environment info
+const deviceId = `${os.hostname()}-${Math.random().toString(36).substring(2, 8).toUpperCase()}`;
+const workspaceId = process.env.ASKUI_WORKSPACE_ID || 'default-workspace-id';
+
+// Define environment info by creating function in allure. 
+allure.writeEnvironmentInfo({
```

Comment:
Can we move this to the `beforeAll`-block? Try to avoide global declarations

Reply:
done. like this - 
<img width="483" alt="image" src="https://github.com/user-attachments/assets/6977dbb1-3b0d-4329-ae07-603382dbda2b">